app/app.js

```diff
@@ -0,0 +1,4 @@
+const express = require("express");
+const app = express();
```

Comment:
I don't see any profit in dividing it into modules. For me, it should be merged with `index.js`

Reply:
merged app.js with index.js